migrations/2024-02-20-093804_ai_help_metadata/up.sql

```diff
@@ -0,0 +1,30 @@
+CREATE TYPE ai_help_message_status AS ENUM (
+    'success',
+    'search_error',
+    'open_ai_api_error',
+    'completion_error',
+    'moderation_error',
+    'no_user_prompt_error',
+    'token_limit_error',
+    'stopped',
+    'timeout',
+    'unknown'
+);
+
+CREATE TABLE ai_help_message_meta (
+    id                  BIGSERIAL PRIMARY KEY,
+    user_id             BIGSERIAL REFERENCES users (id) ON DELETE CASCADE,
+    chat_id             UUID NOT NULL,
+    message_id          UUID NOT NULL,
+    parent_id           UUID DEFAULT NULL REFERENCES ai_help_message_meta (message_id) ON DELETE CASCADE,
+    created_at          TIMESTAMP NOT NULL DEFAULT now(),
+    search_duration     BIGINT NOT NULL,
+    response_duration   BIGINT NOT NULL,
+    query_len           BIGINT NOT NULL,
+    context_len         BIGINT NOT NULL,
+    response_len        BIGINT NOT NULL,
```

Comment:
Need to define what unit to use (characters/bytes/tokens). Maybe characters/bytes is sufficient for query_len and response_len, and context_len might make more sense in tokens. 🤔

Reply:
As discussed, we settle on unit for all, i.e. bytes (what `.len()` returns).